### PR TITLE
Add a workaround for latest Bitcoin Core

### PR DESCRIPTION
**Pull request by romanz**
See https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/190.

### CODE REVIEW

**Comment by Kixunil** [comment]
Does this incompatibility affect master? (will review more properly later)

**Comment by romanz** [comment]
I think it should not:
https://github.com/romanz/electrs/blob/3ef38fc6f1eefa924f1d67273d383f0e5578374a/src/daemon.rs#L98-L106

File: src/daemon.rs [hunk 1]
```diff
@@ -82,8 +96,8 @@ impl Daemon {
         if !network_info.network_active {
             bail!("electrs requires active bitcoind p2p network");
         }
-        let blockchain_info = rpc.get_blockchain_info()?;
-        if blockchain_info.pruned {
+        let info: BlockchainInfo = rpc.call("getblockchaininfo", &[])?;
+        if info.pruned {
```
Comment (Kixunil) [inline src/daemon.rs:100]: Nit: keeping the same name would make diff easier to understand.

**Review by Kixunil — APPROVED** [review]
Looks good

**Pull request merged**